distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -288,7 +289,7 @@ type Number
          Check if 1 is equal to 1.0000001 within 0.001.
 
              1.equals 1.0000001 epsilon=0.001
-    equals : Number -> Number -> Boolean
+    equals : Number -> Number -> Boolean ! Incomparable_Values
     equals self that epsilon=0.0 =
         (self == that) || ((self - that).abs <= epsilon)
```

Comment:
Are you sure about this signature?

When can this yield `Incomparable_Values`? I assume only Numbers will go in here.

I guess with `NaN`? Is `NaN` the **only** possibility of getting `Incomparable_Values` or are other situations that I did not predict?

I'd appreciate a comment explaining the `! Error Conditions` here.